packages/material-ui-lab/src/Skeleton/Skeleton.js

```diff
@@ -47,6 +47,7 @@ export const styles = theme => ({
 
 const Skeleton = React.forwardRef(function Skeleton(props, ref) {
   const {
+    animationDelay,
```

Comment:
Should we change the default value? I imagine it could be interesting not to animate if the skeleton is shown for a short period of time.

Reply:
Im not sure i understand the question :) but ill try

I think it would be a nice ui experience to have a minimum delay of, lets say 0.5s before starting the animation.

Reply:
Yes, I share the same impression. This could help with UI stability while allowing, for long tasks, a feeling of responsiveness. 

However, I'm wondering if the default value shouldn't be set with CSS to avoid specificity issues of inline style.

Reply:
agree, i have added to the class

Reply:
Awesome